src/ranges.js

```diff
@@ -445,3 +445,32 @@ export const lastBufferedEnd = function(a) {
 
   return a.end(a.length - 1);
 };
+
+export const timeAheadOf = function(range, startTime) {
+  let time = 0;
+
+  if (!range || !range.length || !range.end) {
```

Comment:
Is `!range.end` necessary?

Reply:
not sure we do it in other functions in this file.

Reply:
Maybe it was a mistake? I think `end` is a method.